src/ui/public/agg_types/metrics/lib/sibling_pipeline_agg_controller.js

```diff
@@ -0,0 +1,31 @@
+import safeMakeLabel from './safe_make_label';
+
+const siblingPipelineAggController = function (type) {
+  return function ($scope) {
+
+    $scope.aggType = type;
+    //$scope.aggParam = $scope.agg.params[type];
+    $scope.aggTitle = type === 'customMetric' ? 'Metric' : 'Bucket';
+    $scope.aggGroup = type === 'customMetric' ? 'metrics' : 'buckets';
+    $scope.safeMakeLabel = safeMakeLabel;
+
+    $scope.rejectAgg = function (agg) {
```

Comment:
Is this used at all?  I'm a little confused at the interaction and differences between rejectAgg and the aggFilter.  Since the aggFilter already contains all these, if I select for example `Top Hits`, I don't see any of the sibling  drop downs.